hvplot/util.py

```diff
@@ -730,7 +731,7 @@ def import_datashader():
     return datashader
 
 
-def import_geoviews() -> types.ModuleType | None:
+def import_geoviews() -> typing.Union[types.ModuleType, None]:
```

Comment:
An alternative is `from __future__ import annotations`

Reply:
The issue was, I think, with `|` not being supported on Python 3.9. I don't think that would be solved by `from __future__ import annotations`?

```
    def import_geoviews() -> types.ModuleType | None:
E   TypeError: unsupported operand type(s) for |: 'type' and 'NoneType'
```

Reply:
I think it does backport; I remember doing it before if my memory is correct :P
https://www.reddit.com/r/learnpython/comments/14jsyj0/from_future_import_annotation_behavior/

Reply:
> The annotations feature doesn't do anything other than implicitly convert all annotations into strings, meaning there's nothing for the interpreter to evaluate, so even invalid operator use would stop raising errors. The binary OR operator is no exception.

Oh maybe it's just stringifying it?

Reply:
Removed the type hints, let's add them in another PR with mypy testing.